libs/basics/README.md

```diff
@@ -0,0 +1,10 @@
+# basics
+
+An extended TS standard library of generic utility functions and types.
+
+This library should only include utilities that are not specific to VotingWorks
+products, but would be generally applicable to any TypeScript project (similar
+to a library like lodash). In other words, it should be considered as an
+extension of the standard TS libraries. This means it shouldn't have very many
```

Comment:
Maybe even mandate that it has no workspace dependencies so that everyone can import it without fear of cycles?

Reply:
Good call, updated